invenio_cli/helpers/commands.py

```diff
@@ -151,6 +151,23 @@ def _ensure_containers_running(self):
         docker_helper.start_containers()
         # TODO: Find faster way to procede when containers are ready
         time.sleep(30)  # Give time to the containers to start properly
+    
+    def destroy(self):
+        """Destroys env, containers."""
```

Comment:
minor (self-OCD): Since there is space until the 80 characters let's be a bit more specific:
```suggestion
        """Destroys a virtualenv (if created with pipenv) and containers."""
```

Check if it fits in the 80 chars :)